src/Experimental/isomorphism.jl

```diff
@@ -69,20 +84,24 @@ false
 function has_induced_subgraphisomorph(g1::AbstractGraph, g2::AbstractGraph;
                                  vertex_relation::Union{Nothing, Function}=nothing,
                                  edge_relation::Union{Nothing, Function}=nothing,
-                                 alg=:vf2)::Bool
-    if alg == :vf2
+                                 alg::IsomorphismAlgorithm=VF2())::Bool
+
+        has_induced_subgraphisomorph(alg, g1,g2; vertex_relation=vertex_relation, edge_relation=edge_relation)
+
+end
+
+function has_induced_subgraphisomorph(alg,g1::AbstractGraph, g2::AbstractGraph;
```

Comment:
Looking at `base/sort.jl` again, I think the Julia convention  would be to have `alg` as the third argument and not as the first one.

Furthermore, now we both have a signature where `alg` is a regular argument and one, where it is a keyword argument. I think we can delete the one with the keyword argument and instead use a function that does not have `alg` as an argument. 
I.e. we have the signatures
```
has_induced_subgraphisomorph(g1::AbstractGraph, g2::AbstractGraph; kwargs...)
has_induced_subgraphisomorph(g1::AbstractGraph, g2::AbstractGraph, alg; kwargs...)
```